### ISSUE TITLE
Lock object near control buttons

### ISSUE THREAD

**Issue by glixx** [kaa-ching/tbe#10]
If to put any object near buttons "Reset, Pause, Play, Forward", then it cannot be moved or deleted.

**Comment by glixx** [comment]
It's bug. Workaround is only to run level again.
Maybe Milestone A instead of B?

**Comment by kaa-ching** [comment]
Adding it to Milestone A will require that you at least come up with a proposal on how to fix it - I right now have no idea what could be done to prevent this. Suggestions?

**Comment by glixx** [comment]
Is Box2D issue? If yes, then to find steps to reproduce and to open issue http://www.box2d.org/

**Comment by kaa-ching** [comment]
No, this is not a Box2D issue, it's a problem of me not understanding Qt well enough...

**Comment by glixx** [comment]
If to move all control buttons near Info and Eject buttons, then problem will be solved.

**Comment by kaa-ching** [comment]
> If to move all control buttons near Info and Eject buttons, then problem will be solved.

Not gonna happen. I like the buttons where they are.

Another option would be to do this in concert with the remaining part of issue #91.

**Comment by kaa-ching** [comment]
as agreed between @lspacenl, @amarsman : move to next milestone.
